src/Microsoft.Azure.WebJobs.Host/Bindings/Runtime/Binder.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Most of the logic in this file was factored out to here from `RuntimeValueProvider` as you'll see below